src/pages/Users/Login.js

```diff
@@ -1,11 +1,9 @@
-import React, { Component } from 'react'
-import AuthService from '../../services/AuthService'
-import { Redirect } from 'react-router'
+import React, { Component } from 'react';
```

Comment:
I'm approving the PR, but in another PR, I recommend renaming these components with more specificity.  This component looks like it primarily hosts a login form ?  Mayhaps, `LoginForm` would be a better name.